charts/openebs/network/experiments/openebs-pool-faliure.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Openebs pool failure check !
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: kubernetes
+  name: openebs-pool-failure
+  namespace: percona1
+spec:
+  definition:
+    args:
+    - -c
+    - ansible-playbook ./experiments/chaos/openebs_pool_failure/test.yml -i /etc/ansible/hosts
+      -vv; exit 0
+    command:
+    - /bin/bash
+    - name: ANSIBLE_STDOUT_CALLBACK
+      value: default
```

Comment:
Put one blank line space between each value pair, And write a comment for the optional value!!